### DISCUSSION TITLE
Welcome to FreeControl Discussions!

### DISCUSSION THREAD

**Discussion by pdone** [pdone/FreeControl#56] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by cc77077** [comment]
我的连接就过一会退出了怎么整（纯小白）

**Reply by cc77077** [reply]
[Uploading 2025-01-03.log…]()

**Reply by cc77077** [reply]
这是日志

**Reply by ghost** [reply]
能连接，但是卡，也没办法控制手机端
